samples/employee-portal-cell/sample-hr-cell.yaml

```diff
@@ -12,13 +12,21 @@ spec:
       ingress:
         http:
         - context: /info
+          global: true
+          version: 5.0.0
+          authenticate: true
           definitions:
           - path: /
             method: GET
           port: 80
           destination: 
             host: hr
             port: 80
+        extensions:
+          apiPublisher:
+            authenticate: true
```

Comment:
What is the reason to have the authenticate flag here? If we have it, then it should override the same flag in ingreee.http section.

Reply:
This flag overrides the API level authentication